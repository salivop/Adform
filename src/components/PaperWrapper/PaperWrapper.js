import React from "react";
import Paper from "@material-ui/core/Paper";

import { styles } from "./PaperWrapper.styles";

const PaperWrapper = ({ child }) => <Paper style={styles.paper}>{child}</Paper>;

export default PaperWrapper;
