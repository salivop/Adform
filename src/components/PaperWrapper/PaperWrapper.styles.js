export const styles = {
  paper: {
    top: "50%",
    position: "absolute",
    transform: "translate(-50%, -50%)",
    left: "50%",
    padding: "30px",
  }
};
