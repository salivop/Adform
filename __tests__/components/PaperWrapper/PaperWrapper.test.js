import React from "react";
import toJson from 'enzyme-to-json';
import { shallow } from "enzyme";

import PaperWrapper from "src/components/PaperWrapper/PaperWrapper";

describe("PaperWrapper", () => {
  let wrapper;

  beforeEach(() => (wrapper = shallow(<PaperWrapper />)));

  it("should render correctly", () => {
    const component = wrapper;
    expect(toJson(component)).toMatchSnapshot();
  });
});
