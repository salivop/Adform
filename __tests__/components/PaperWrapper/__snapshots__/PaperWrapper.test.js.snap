// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyComponent should render correctly 1`] = `
<WithStyles(ForwardRef(Paper))
  style={
    Object {
      "left": "50%",
      "padding": "30px",
      "position": "absolute",
      "top": "50%",
      "transform": "translate(-50%, -50%)",
    }
  }
/>
`;

exports[`PaperWrapper should render correctly 1`] = `
<WithStyles(ForwardRef(Paper))
  style={
    Object {
      "left": "50%",
      "padding": "30px",
      "position": "absolute",
      "top": "50%",
      "transform": "translate(-50%, -50%)",
    }
  }
/>
`;
